### PR TITLE
upgrade to Nextra v2

### PR DESCRIPTION
**Pull request by promer94**
using `Tabs` `filename` and `copy` utils from nextra 

Screenshot 2022-12-05 at 17 04 27

### CODE REVIEW

File: pages/docs/configuration/modules.mdx [hunk 1]
```diff
@@ -118,17 +121,9 @@ The value of `lazy` has a few possible effects:
 - `true` - Do not lazy-initialize local `./foo` imports, but lazy-init `foo` dependencies. Local paths are much more likely to have circular dependencies, which may break if loaded lazily, so they are not lazy by default, whereas dependencies between independent modules are rarely cyclical.
 - `Array<string>` - Lazy-initialize all imports with source matching one of the given strings.
 
-The two cases where imports can never be lazy are:
```
Comment (kdy1) [inline pages/docs/configuration/modules.mdx:121]: Why did you delete this?

File: pages/docs/configuration/modules.mdx [hunk 2]
```diff
@@ -98,11 +98,14 @@ These options are shared by `commonjs` / `es6` / `umd` / `amd` inside `.swcrc`:
 }
 ```
 
+{/* TODO: should fix this in nextra side */}
+export const EsModuleText = () => <code className="nx-border-black nx-border-opacity-[0.04] nx-bg-opacity-[0.03] nx-bg-black nx-break-words nx-rounded-md nx-border nx-py-0.5 nx-px-[.25em] nx-text-[.9em] dark:nx-border-white/10 dark:nx-bg-white/10">{`__esModule`}</code>
```
Comment (promer94) [inline pages/docs/configuration/modules.mdx:102]: https://github.com/vercel/next.js/issues/43732
